compiler/lib/Dialect/mhlo/Transforms/GenericFusion.cpp

```diff
@@ -97,8 +97,8 @@ bool isFusibleWith(Operation *target, Operation * /*start*/) {
   return target->hasTrait<::mlir::OpTrait::Elementwise>() ||
          target->hasTrait<hlo::OpTrait::BroadcastingElementwise>() ||
          isSplatMhloConstantLike(target) ||
-         isa<mhlo::BroadcastInDimOp, mhlo::BroadcastOp, mhlo::ReshapeOp>(
-             target) ||
+         isa<mhlo::BroadcastInDimOp, mhlo::BroadcastOp, mhlo::ReshapeOp,
+             mhlo::TransposeOp>(target) ||
```

Comment:
should not fuse tranpose？

Reply:
> should not fuse tranpose？

There are `unsupported transpose permutation` runtime error with horizontal fusion. We need to support these transpose patterns

Reply:
I'd like to fix this later